### PR TITLE
musescore: bump to 3.6.2

### PR DESCRIPTION
**Pull request by kenmays**
MuseScore 3.6.2 (February 8, 2021)

### CODE REVIEW

**Comment by Begasus** [comment]
Need some checks, was able to build previous with extra option "-DCMAKE_SKIP_RPATH=ON" and run it on 32bit :)

**Comment by kenmays** [comment]
Confirmed on x86:creating package musescore_x86-3.6.2-1-x86_gcc2.hpkg ...
```
----- Package Info ----------------
header size:                     80
heap size:                231806099
TOC size:                     26255
package attributes size:       1035
total size:               231806179
-----------------------------------
waiting for build package musescore_x86-3.6.2-1 to be deactivated
grabbing musescore_x86-3.6.2-1-x86_gcc2.hpkg and moving it to /boot/home/haikuports/packages/musescore_x86-3.6.2-1-x86_gcc2.hpkg
```

File: media-sound/musescore/musescore-3.6.2.recipe [hunk 1]
```diff
@@ -71,7 +71,7 @@ BUILD()
 	make PREFIX=$appsDir/MuseScore UPDATE_CACHE=FALSE BUILD_PULSEAUDIO=OFF \
 		BUILD_JACK=OFF BUILD_PORTMIDI=OFF BUILD_PORTAUDIO=OFF BUILD_WEBENGINE=OFF \
 		USE_SYSTEM_FREETYPE=ON DOWNLOAD_SOUNDFONT=OFF BUILD_ALSA=OFF \
-		MUSESCORE_BUILD_CONFIG=release revision release
+		NO_RPATH=TRUE MUSESCORE_BUILD_CONFIG=release revision release
```
Comment (Begasus) [inline media-sound/musescore/musescore-3.6.2.recipe:74]: Would be nice if you would add the separate option on a new line, makes it easier to follow :)
eg; 
```diff
- 		NO_RPATH=TRUE MUSESCORE_BUILD_CONFIG=release revision release
+ 		NO_RPATH=TRUE \
+		MUSESCORE_BUILD_CONFIG=release revision release
```
Also for the previous lines

**Comment by Begasus** [comment]
Should be ok, keep an eye out on the buildbot :) thanks!

**Pull request merged**